sdk/core/azure-core/src/samples/java/com/azure/core/util/paging/PagedFluxCoreJavaDocCodeSnippets.java

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.util.paging;
+
+import com.azure.core.util.IterableStream;
+import reactor.core.publisher.Flux;
+
+import java.util.function.Supplier;
+
+/**
+ * Code snippets for {@link ContinuablePagedFluxCore}
+ */
+public class PagedFluxCoreJavaDocCodeSnippets {
+    /**
+     * Code snippets for extending from {@link ContinuablePagedFluxCore} and enabling custom continuation token.
+     */
+    public void customContinuationTokenSnippet() {
+        // BEGIN: com.azure.core.util.paging.pagedfluxcore.continuationtoken
+        class ContinuationState<C> {
+            private C lastContinuationToken;
+            private boolean isDone;
+
+            ContinuationState(C token) {
+                this.lastContinuationToken = token;
+            }
+
+            void setLastContinuationToken(C token) {
+                this.isDone = token == null ? true : false;
```

Comment:
This can be simplified to: `this.isDone = token == null;`

Reply:
yes, updated the PR.